mrmustard/math/backend_jax.py

```diff
@@ -748,7 +750,7 @@ def hermite_renormalized_1leftoverMode_reorderedAB(
         function = partial(hermite_multidimensional_1leftoverMode, cutoffs=tuple(cutoffs))
```

Comment:
Shouldn't we also update the signature of `hermite_renormalized_1leftoverMode_reorderedAB` to match `hermite_renormalized_1leftoverMode`? 

Also, doesn't the partial call to function need to be updated here? I.e. 

```suggestion
        function = partial(hermite_multidimensional_1leftoverMode, output_cutoff = cutoffs[0], pnr_cutoffs=cutoffs[1:])
```

Reply:
Good catch for partial, it must be something the tests are not covering. Regarding `hermite_renormalized_1leftoverMode_reorderedAB`, as soon as I'll write the backward pass of the new diagonal algorithm we won't need it anymore (and we can delete the compactFock stuff entirely), so I didn't want to spend time on it.